nabs/tests/test_plans.py

```diff
@@ -69,7 +69,7 @@ def test_delay_scan(RE, hw, time_motor):
 
     # Speed of light is more or less 3e8
     goal = 1/(3e8)
-    msgs = nbp.delay_scan([hw.det], time_motor, [0, goal], 1, duration=0.01)
+    msgs = nbp.delay_scan([hw.det], time_motor, [0, goal], 1, duration=0.1)
```

Comment:
Stuff like this makes me want to refactor delay_scan to make it more testable.

Reply:
Yeah I was thinking about how we might end up mocking these better, rather than just letting the global time tick.  But with the simulated motors I'm not really sure how that might look